provider/utils/init_provider.go

```diff
@@ -86,26 +86,28 @@ func isEmptyStringValue(value *string) bool {
 }
 
 // OpenProviderSession ...
-func OpenProviderSession(conf *config.Config, providers registry.Providers, providerID string, logger *zap.Logger) (session provider.Session, fatal bool, err1 error) {
+func OpenProviderSession(conf *config.Config, providers registry.Providers, providerID string, logger *zap.Logger) (session provider.Session, fatal bool, errReturn error) {
 	prov, err := providers.Get(providerID)
```

Comment:
we do not need errReturn and err separately.
We can assign to err itself,

Reply:
errReturn is for returning from the method, and we can't use  errReturn instead of err

Reply:
https://github.com/IBM/ibmcloud-storage-volume-lib/blob/master/volume-providers/vpc/provider/create_volume.go#L27-L30

Reply:
It is not required.

Reply:
> It is not required.

Done